custom_components/tuya_local/devices/ballu_aura.yaml

```diff
@@ -0,0 +1,104 @@
+name: Ballu Aura
+products:
+  - id: r9lwbruavixdspeb
+    name: "Ballu Aura  9/12"
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: "COOL"
+              value: cool
+            - dps_val: "DRY"
+              value: dry
+            - dps_val: "FAN"
+              value: fan_only
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 17
+        max: 30
+      unit: C
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: mode
+      type: string
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: "1"
+          value: low
+        - dps_val: "2"
+          value: medium
+        - dps_val: "3"
+          value: high
+    - id: 19
+      name: temperature_unit
```

Comment:
I think it is better to leave this out of here unless you know how the temperature unit setting functions. Often devices use different dps for F, or at least the temperature range will need adjusting when it is set to F. But in some devices it only affects the display, in which case setting the unit at entity level is wrong.